### PR TITLE
Update all_sky tests

### PR DESCRIPTION
**Pull request by sriharshakandala**
Update all-sky tests to be able to use user prescribed number of columns

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by sriharshakandala** [comment]
bors try

File: test/all_sky_utils.jl [hunk 1]
```diff
@@ -41,19 +42,16 @@ function all_sky(
...
 
     input_file = get_ref_filename(:atmos_state, :cloudysky)                    # all-sky atmos state
-    if use_lut
-        flux_file = get_ref_filename(:comparison, :cloudysky, lut_pade = :lut) # flux files for comparison (LUT)
-    else
-        flux_file = get_ref_filename(:comparison, :cloudysky, lut_pade = :pade) # flux files for comparison (pade)
-    end
+    flux_file =
+        use_lut ? get_ref_filename(:comparison, :cloudysky, lut_pade = :lut) : # flux files for comparison (LUT)
+        get_ref_filename(:comparison, :cloudysky, lut_pade = :pade) # flux files for comparison (pade)
```
Comment (charleskawczynski) [inline test/all_sky_utils.jl:54]:
```suggestion
    lut_pade = use_lut ? :lut : :pade # flux files for comparison (LUT/pade)
    flux_file = get_ref_filename(:comparison, :cloudysky, lut_pade)
```

File: test/read_all_sky.jl [hunk 2]
```diff
@@ -1,3 +1,10 @@
+function ncol_ds_all_sky(use_lut)
+    flux_file =
+        use_lut ? get_ref_filename(:comparison, :cloudysky, lut_pade = :lut) : # flux files for comparison (LUT)
+        get_ref_filename(:comparison, :cloudysky, lut_pade = :pade)  # flux files for comparison (pade)
```
Comment (charleskawczynski) [inline test/read_all_sky.jl:4]:
```suggestion
    # flux files for comparison (LUT/pade)
    lut_pade = use_lut ? :lut : :pade
    flux_file = get_ref_filename(:comparison, :cloudysky, lut_pade)
```

File: test/read_all_sky.jl [hunk 3]
```diff
@@ -183,3 +191,24 @@ function setup_allsky_as(
         bot_at_1,
     )
 end
+
+_orient_data(data, bot_at_1) = bot_at_1 ? data : reverse(data, dims = 1)
+
+function load_comparison_data(use_lut, bot_at_1, ncol)
+    flux_file =
+        use_lut ? get_ref_filename(:comparison, :cloudysky, lut_pade = :lut) : # flux files for comparison (LUT)
+        get_ref_filename(:comparison, :cloudysky, lut_pade = :pade)  # flux files for comparison (pade)
```
Comment (charleskawczynski) [inline test/read_all_sky.jl:200]:
```suggestion
    # flux files for comparison (LUT/pade)
    lut_pade = use_lut ? :lut : :pade
    flux_file = get_ref_filename(:comparison, :cloudysky, lut_pade)
```

**Review by charleskawczynski — APPROVED** [review]
I made a suggestion to simplify the if-else block, the code changes otherwise look good!

**Pull request merged**